packages/web-components/src/components/content-section/__stories__/content-section.stories.react.tsx

```diff
@@ -0,0 +1,150 @@
+
+For example: optical scan of documents (to create a text file out of an image of text) used to be considered artificial intelligence before it became common in our everyday lives. Observers of the history of AI call this phenomenon "the AI effect."
+`;
+
+const card1 = (
+  <DDSContentGroupCardsItem href="https://www.example.com">
+    <DDSCardHeading>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt</DDSCardHeading>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+    </p>
+    <DDSCardFooter icon-placemenet="left">
+      <ArrowRight20 slot="icon" />
+    </DDSCardFooter>
+  </DDSContentGroupCardsItem>
+);
+
+const card2 = (
+  <DDSContentGroupCardsItem href="https://www.example.com">
+    <DDSCardHeading>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt</DDSCardHeading>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
+    <DDSCardFooter icon-placemenet="left">
+      <ArrowRight20 slot="icon" />
+    </DDSCardFooter>
+  </DDSContentGroupCardsItem>
+);
+
+export const Default = ({ parameters }) => {
+  const { heading, copy, addChildren } = parameters?.props?.ContentSection ?? {};
+  return (
+    <DDSContentSection>
+      <DDSContentSectionHeading>{heading}</DDSContentSectionHeading>
+      <DDSContentSectionCopy>{copy}</DDSContentSectionCopy>
+      {addChildren.includes('Content block simple') ? (
+        <DDSContentBlockSimple>
+          <DDSContentBlockHeading>What's the latest news in artificial intelligence?</DDSContentBlockHeading>
+          <DDSContentBlockCopy size="sm">{blockCopy}</DDSContentBlockCopy>
+          <DDSVideoPlayerContainer slot="media" video-id="1_9h94wo6b"></DDSVideoPlayerContainer>
+          <DDSTextCTA slot="footer" cta-type="jump" href="https://www.ibm.com">
+            Jump to AI ethics and trust
+          </DDSTextCTA>
+        </DDSContentBlockSimple>
+      ) : (
+        ''
+      )}
+      {addChildren.includes('Content group cards') ? (
+        <DDSContentGroupCards>
+          <DDSContentGroupHeading>Lorem ipssum dolor sit amet.</DDSContentGroupHeading>
+          <DDSContentGroupCopy>Lorem ipsum dolo sit amet.</DDSContentGroupCopy>
+          {card1}
+          {card2}
+          {card1}
+          {card2}
+        </DDSContentGroupCards>
+      ) : (
+        ''
+      )}
+      <DDSTextCTA slot="footer" cta-type="local" href="https://www.example.com">
+        Link action
+      </DDSTextCTA>
+    </DDSContentSection>
+  );
+};
+
+export default {
+  title: 'Components/Content section',
+  decorators: [
+    story => {
+      return (
+        <>
+          <div className="bx--grid">
+            <div className="bx--row">
+              <div className="bx--col-lg-12 bx--no-gutter">
```

Comment:
Grid columns here don't match the [web components](https://www.ibm.com/standards/carbon/web-components/?path=/story/components-content-section--default) version.